joomla/joomla-cms
# [4.0] Site Contacts Category List Search Filters

### Summary of Changes
Adding Search Filter and Clear buttons to the `List Contacts in a Category` menu item.
Modify the placeholder string to specify the search is limited to the Contact Title.


### Testing Instructions
Create multiple contacts in the same category.
Create a `List Contacts in a Category` menu item for this category.
Make sure Filter Field is set to `Show` in the menu parameters.
Display the menu item in frontend.


### Actual result BEFORE applying this Pull Request

<img width="898" alt="Screen Shot 2020-09-30 at 12 53 06" src="https://user-images.githubusercontent.com/869724/94676632-ef605400-031b-11eb-9d20-76da5ddef119.png">


### Expected result AFTER applying this Pull Request
![contact-search](https://user-images.githubusercontent.com/869724/94676231-4b76a880-031b-11eb-8788-4087d66e0af1.gif)



### Note
1. With a similar PR, we can add a Clear filter button in the `List Articles in a Category` menu item
2. It would be nice to add css for the Clear button in order to display a different color when the filter field is empty and not. I guess this would be for the Cassiopea Team to deal with.
